cards/acute-limb-ischemia/card.md

```diff
@@ -34,16 +34,16 @@ categories:
 - **Neurological:** Check for sensory and motor loss
 - **Vascular:** Grade peripheral pulses
 
-- 0 = No Doppler signal
-- 1 = Reduced
-- 2 = Normal
-- 3 = Increased
-- 4 = Bounding
+    - 0 = No Doppler signal
```

Comment:
Oh! I misread this. I see the nested list items. Sorry.

I'm not sure our build rules allow it but we can tweak.